Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -0,0 +1,306 @@
+using System.Collections.Generic;
+using UnityEngine;
+#if AT_USE_SPLINES
+using UnityEngine.Splines;
+using Unity.Mathematics;
+#endif
+
+#if AT_USE_PROBUILDER && UNITY_EDITOR
+using UnityEngine.ProBuilder;
+#endif
+
+namespace Unity.MeshSync
+{
+    partial class MeshSyncServer
+    {
+        public enum PropertiesState
+        {
+            None,
+            Sending,
+            Received
+        }
+
+        bool needsClientSync;
+
+        Dictionary<EntityRecord, Matrix4x4> entityTransforms = new Dictionary<EntityRecord, Matrix4x4>();
+
+#if UNITY_EDITOR
+        public override InstanceHandlingType InstanceHandling
+        {
+            get => base.InstanceHandling; 
+            set
```

Comment:
If we just need to override the set, wouldn't it better to convert this into a common getter and a virtual setter methods ?

Reply:
We can't have virtual setters only, the whole property needs to be virtual.

Reply:
Personally I'd prefer properties but don't mind either way.